### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Evilcarbon** [codex-team/editor.js#2440] [Q&A]
problem fixed

**Comment by neSpecc** [comment]
If you don't want to use bundler, load Editor.js from CDN instead of `/node_modules/` because the browser can't resolve it.

**Reply by Evilcarbon** [reply]
But am unable to even use the CDN method. Yes am new to Js  and I really don't know how to get this running in the first place. The only examples I see online is using the node_modules. Please can you help?

**Reply by Evilcarbon** [reply]
Hello @neSpecc can you help?

**Reply by neSpecc** [reply]
Add this script to your page:

```html
<script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
```

**Reply by Evilcarbon** [reply]
Yeah I saw this also on the website and sorry to ask but how will the Js look like since I can't use import in the case of CDN

**Reply by doldesh** [reply]
> Yeah I saw this also on the website and sorry to ask but how will the Js look like since I can't use import in the case of CDN

Did you find a solution? @Evilcarbon 

I'm currently getting the same errors and I'm stuck.

**Reply by neSpecc** [reply]
Check the example https://github.com/codex-team/editor.js/blob/next/example/example.html
